### PR TITLE
Triage 2024 03 19

### PR DESCRIPTION
**Pull request by Kobzol**
[Rendered](https://github.com/Kobzol/rustc-perf/blob/9b4df43b82c5d0fd214d6fae1b8ba4f5e3fdfec1/triage/2024-03-19.md).

### CODE REVIEW

File: triage/2024-03-19.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+|  Improvements ✅ <br /> (primary)  |   -   |       -        |   0   |
+| Improvements ✅ <br /> (secondary) | -1.3% | [-1.3%, -1.3%] |   1   |
+|         All ❌✅ (primary)          | 2.0%  |  [0.2%, 7.8%]  |  193  |
+
+* This PR adds additional verification to the compiler that checks if incremental compilation didn't
+  encounter problems.
+* This result looks quite bad, but that is mostly caused by the configuration of our benchmark
+  suite,
+  which fully enables this verification to make benchmarking more deterministic.
+* Normal builds of user code should only pay 1/32 of the cost here, so it should not be an issue.
```
Comment (lqd) [inline triage/2024-03-19.md:73]: I had the same misconception, but I believe is wrong. My understanding (which could be wrong) is: the checks didn’t change themselves, but new checks were added under the same feature gate, which rustc-perf enables. I don’t believe the checks are done outside of debug-assertions, and the "force incremental verification checks" opt-in, i.e. not present in the 1/32nd of all query calls.

Comment (Kobzol) [reply triage/2024-03-19.md:73]: Ah, I see, makes sense. So in other words, it shouldn't be a change for normal Rust code.

Comment (Kobzol) [reply triage/2024-03-19.md:73]: Ah, I see, makes sense. So in other words, it shouldn't be a change for normal Rust code.

Comment (Kobzol) [reply triage/2024-03-19.md:73]: Ah, I see, makes sense. So in other words, it shouldn't be a change for normal Rust code.

Comment (lqd) [reply triage/2024-03-19.md:73]: I believe so, from what I remember of the PR

Comment (Kobzol) [reply triage/2024-03-19.md]: Ok, reworded.

**Review by lqd — APPROVED** [review]

**Pull request merged**